### PR TITLE
feat: add linkedin color code and change my data

### PR DESCRIPTION
**Pull request by RoshanMhatre**
add Linkedin color code and changed my data

## Proposed Changes
Changed twitter link and content data of roshanmhatre
add Linkedin color code

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/193"></a>

### CODE REVIEW

File: public/data/_list.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "avatar": "https://github.com/Skyhero-admin.png"
+  },
+  { "username": "krishdevdb", "avatar": "https://github.com/krishdevdb.png" 
+  },
+  { "username": "vinzvinci", "avatar": "https://github.com/vinzvinci.png" },
+  { "username": "voidz", "avatar": "https://github.com/voidz7.png" },
+  { "username": "suhail34", "avatar": "https://github.com/suhail34.png" },
+  { "username": "Rajiv Tiwari", "avatar": "https://github.com/rajivransom.png" },
+  { "username": "RoshanMhatre", "avatar": "https://github.com/RoshanMhatre.png" }
+]
```
Comment (FayasNoushad) [inline public/data/_list.json:29]: This file not needed, You can delete this.

File: public/list.json [hunk 2]
```diff
@@ -0,0 +1 @@
...ma254","avatar":"https://github.com/orama254.png"},{"username":"ragibajmal3","avatar":"https://github.com/RagibAjmal.png"},{"username":"recognizeyourself","avatar":"https://github.com/Recognizeyourself.png"},{"username":"rishabhrathore055","avatar":"https://github.com/rishabhrathore055.png"},{"username":"roshanmhatre","avatar":"https://github.com/RoshanMhatre.png"},{"username":"SahityaRoy","avatar":"https://github.com/SahityaRoy.png"},{"username":"ShaileshKumar007","avatar":"https://github.com/shaileshkumar007.png"},{"username":"Skyhero-admin","avatar":"https://github.com/Skyhero-admin.png"},{"username":"sohamsattigeri","avatar":"https://avatars.githubusercontent.com/u/79072777?v=4"},{"username":"SriSatyaT","avatar":"https://github.com/SriSatyaT.png"},{"username":"stemount","avatar":"https://github.com/stemount.png"},{"username":"suhail34","avatar":"https://github.com/suhail34.png"},{"username":"verma-kunal","avatar":"https://github.com/verma-kunal.png"},{"username":"vikasganiga05","avatar":"https://github.com/vikasganiga05.png"},{"username":"vinzvinci","avatar":"https://github.com/vinzvinci.png"},{"username":"voidz","avatar":"https://avatars.githubusercontent.com/u/62803658?v=4"}]
```
Comment (FayasNoushad) [inline public/list.json:1]: This file also not needed.

File: public/data/_list.json (hunk 1 above)
Comment (RoshanMhatre) [reply public/data/_list.json:29]: Should I delete only mine or others too?

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
This looks good to me, however the `public/list.json` file should be removed(as mentioned above)

**Comment by eddiejaoude** [comment]
I will close this PR as it is starting to get lots of conflicts, please can you create another PR when you have time

**Pull request closed without merging**